assets/app/views/directives/duration.html

```diff
@@ -0,0 +1,2 @@
+<span ng-if="round" data-timestamp="{{timestamp}}" class="duration-rounded">{{timestamp | duration : null : true : 1}}</span>
```

Comment:
why not just make it precision all the way through, we might want to let the directive show more precision (hr + min + sec) somewhere at some point